### DISCUSSION TITLE
Launching quicklook by default?

### DISCUSSION THREAD

**Discussion by yosun** [google/model-viewer#4811] [Q&A]
Is it possible to launch quicklook by default upon loading modelviewer?

**Comment by danilokrulj** [comment]
1| Do you mean launch quicklook instead of another viewer?
2| Or do you mean launch quicklook without user interaction (clicking AR button)?


1) QuickLook is the only viewer for iOS at the moment if I am not mistaken, so its the only option for those devices and can't be used on Android devices.

2) I wanted to have this functionality too, but I was told that user interaction is a must, so I guess the answer to this question is also no. But if I remember correctly I did manage to get it working only on iOS (quicklook) but since I couldn't get it to work on Android I gave up on that functionality.

https://github.com/google/model-viewer/discussions/4199

**Comment by yosun** [comment]
hmm what about launching just usdz conversion

**Reply by elalish** [reply]
We haven't exposed this function in our public API, but it would be easy. If you can describe your use-case, a PR is always welcome!

**Reply by yosun** [reply]
modelviewer can grow to become an ultimate swiss army knife of AR - enabling any model to be convertible and Air-droppable with AR features offline!

**Reply by elalish** [reply]
Well, that is more or less how we see the project. What would you do with the generated USDZ besides pass it to Quick Look like we already do? We haven't done this so far because USDZ looks like a dead-end proprietary format (unlike USD) that we'd prefer our users not to have to understand.

**Reply by yosun** [reply]
IMO: AR is still an underdog (and far from being considered mainstream tech)... So, I think it helps the industry more if people are unified: it should be less about platform or file format wars and more about making adoption and discovery by the masses easier. 

On the one hand, "AR" has been around for a long time and we are now in mature SDK land for ARKit/ARCore, but also people outside of tech and the coastal cities haven't seen it. Take a roadtrip across America and show at least 3 people in the boonies about AR. It's like magic to all 3 of them. Which means they've never really seen it before.

**Reply by danilokrulj** [reply]
I think that is the aim here in model-viewer with WebXR, to have equal experiences and to unify users by using one viewer on both iOS and Android but iOS is having difficulties adopting WebXR, thus making the process harder :(

**Reply by elalish** [reply]
Yeah, I agree with everything here. But I'd still like to know what `exportUSDZ()` will do for our users.

**Comment by danilokrulj** [comment]
I'm not really sure I understand. 
You can find usdz conversion code from the three.js library if that's what you need. From what I know, model-viewer uses threes usdz conversion function.
